grid/recruitment/create-requisition/maps/mock.test.ts

```diff
@@ -0,0 +1,33 @@
+import { SuperfaceTest } from '@superfaceai/testing';
+
+import { buildSuperfaceTest } from '../../../test-config';
+
+describe('recruitment/create-requisition/mock', () => {
+  let superface: SuperfaceTest;
+
+  beforeEach(() => {
+    superface = buildSuperfaceTest({
+      profile: 'recruitment/create-requisition',
+      provider: 'mock',
```

Comment:
Why make a special test for mock? A lot of other profiles in station simply use the same generic test.